src/analysis/retail/monk/mistweaver/modules/spells/InvokeChiJi.tsx

```diff
@@ -158,9 +164,22 @@ class InvokeChiJi extends BaseCelestialAnalyzer {
     if (!this.celestialActive) {
       return;
     }
-    if (this.selectedCombatant.getBuffStacks(SPELLS.INVOKE_CHIJI_THE_RED_CRANE_BUFF.id) === 3) {
+    if (!this.castBokInWindow) {
+      this.castBokInWindow = true;
+      return;
+    }
```

Comment:
i know you explained in the pr, but could use an explanatory comment here too
```suggestion
    // the first cast of BoK in the window should overcap (intentionally). ignore it
    if (!this.castBokInWindow) {
      this.castBokInWindow = true;
      return;
    }
```